esp/services/ws_dfu/ws_dfuXRefService.cpp

```
@@ -553,6 +553,9 @@ bool CWsDfuXRefEx::onDFUXRefList(IEspContext &context, IEspDFUXRefListRequest &r
 {
     try
     {
+#ifdef _CONTAINERIZED
+        IERRLOG("CONTAINERIZED(CWsDfuXRefEx::onDFUXRefList)");
```

Why IERRLOG here but UNIMPLEMENTED below?

---

I tried to use UNIMPLEMENTED when the code would have previously failed, and IERRLOG when it would have silently ignored it.